### ISSUE TITLE
DIV LOSS

### ISSUE THREAD

**Issue by Dragon-hu** [JXYin24/MCM#2]
hello, during training  the div loss may be a negative number, how do you deal with it?

**Comment by JXYin24** [comment]
To avoid this issue, you can modify "loss_tensor = torch.log(trans_matrix) * scale" to "loss_tensor = torch.log(1+trans_matrix) * scale"
